scripts/update-input.py

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: 2022 Oxhead Alpha
```

Comment:
Does it make sense for you to add a shebang `#!/usr/bin/env python3` and use this script as an executable?